pkg/api/api.go

```diff
@@ -50,6 +50,8 @@ type ConfigHandler func(actualCfg interface{}, defaultCfg interface{}) http.Hand
 type Config struct {
 	SkipLabelNameValidationHeader bool `yaml:"skip_label_name_validation_header_enabled" category:"advanced"`
 
+	enableOtelMetadataStorage bool `yaml:"enable_otel_metadata_translation" category:"experimental"`
```

Comment:
Could you please add this to the list of experimental features in [the docs](https://github.com/grafana/mimir/blob/main/docs/sources/mimir/configure/about-versioning.md)?

Reply:
will do

Reply:
done, is it ok?